src/index.js

```diff
@@ -1 +1,52 @@
-export default "TODO";
+import * as babylon from "babylon";
+
+// Parse options values.
+const _parseOpts = (opts) => Object.keys(opts)
+  .map((key) =>
+    [key, babylon.parse(`expr=${opts[key]}`).program.body.pop().expression.right])
```

Comment:
This looks very heavy-handed.  Is this in order to support any JS expression type?